awx/main/models/ha.py

```diff
@@ -131,6 +131,13 @@ def is_lost(self, ref_time=None):
         grace_period = 120
         return self.modified < ref_time - timedelta(seconds=grace_period)
 
+    def mark_offline(self, on_good_terms=False):
```

Comment:
I'm still reading through the PR, so don't have the full context yet, but maybe `verified` instead of `on_good_terms`?

Reply:
the distinction is whether it is marked offline due to judgement of non-compliance by a peer, or the node marks itself offline as it conducts a normal shutdown.

So I mean to express that it's an amicable breakup from the cluster in one case. I don't like any of the other namings quite as much.